--- SPECS/automake.spec.orig	2020-05-06 02:48:47.840620663 +0000
+++ SPECS/automake.spec	2020-05-06 05:25:16.474955530 +0000
@@ -2 +2 @@
-%global api_version 1.13
+%global api_version 1.16
@@ -6,2 +6,2 @@
-Version:    %{api_version}.4
-Release: 3%{?_rebuild}%{?dist}
+Version:    %{api_version}.2
+Release: 0%{?_rebuild}.mtf
@@ -72,2 +72,2 @@
-%patch0 -p1 -b .disable_tests
-autoreconf -iv
+#%patch0 -p1 -b .disable_tests
+#autoreconf -iv
